flink-streaming-java/src/main/java/org/apache/flink/streaming/runtime/tasks/SystemProcessingTimeService.java

```diff
@@ -197,6 +204,23 @@ public boolean shutdownAndAwaitPending(long time, TimeUnit timeUnit) throws Inte
 		return timerService.awaitTermination(time, timeUnit);
 	}
 
+	@Override
+	public boolean shutdownServiceUninterruptible(long timeoutMs) {
```

Comment:
Do we want to make the shutdown of the timer service swallow the `InterruptedException` also for all subsequent calls happening after the shutdown of the timer service?

Reply:
👍